src/explore-education-statistics-common/src/modules/charts/components/LineChartLabel.tsx

```diff
@@ -0,0 +1,79 @@
+import { LineChartDataLabelPosition } from '@common/modules/charts/types/chart';
+import { LegendInlinePosition } from '@common/modules/charts/types/legend';
+import formatPretty from '@common/utils/number/formatPretty';
+import { LabelProps } from 'recharts';
+import React from 'react';
+
+interface Props extends LabelProps {
+  colour: string;
+  dataLabelPosition?: LineChartDataLabelPosition;
+  decimalPlaces?: number;
+  index: number;
+  name: string;
+  legendLabelPosition: LegendInlinePosition;
+  showDataLabels?: boolean;
+  showLegendAsLabel?: boolean;
```

Comment:
Would suggest renaming these to `isDataLabel` and `isLegendLabel`.

On a side note, I wouldn't normally encourage having multiple boolean props to express a single outcome like this. The alternative would be to have a `type` prop instead (which would have a value of either `legend` or `data`), but this makes the calling function a little bit messy so I'll allow it this time :laughing: